### PR TITLE
Currency on dialog form

### PR DESCRIPTION
**Pull request by Mussuranae**
Add a currency icon on the investment dialog form.
image

I need suggestion for the getCurrencySymbol function, I have the feeling that it is not the best way but I prefer to use the function instead of doing if statement in the html.

### CODE REVIEW

File: apps/financiers/financiers/src/app/marketplace/title/view/view.component.html [hunk 1]
```diff
@@ -58,10 +58,12 @@ <h2>What do you want to know about this project?</h2>
           <mat-form-field fxFlex appearance="outline">
             <mat-label>From</mat-label>
             <input type="number" matInput formControlName="from" min="0" />
+            <span matPrefix>{{ getCurrencySymbol((campaign$ | async)?.currency, 'wide') }}</span>
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/marketplace/title/view/view.component.html:61]: There is a currency pipe you can use. 
```
 amount | currency: form.getCurrency() : 'symbol' : '1.0-0' 
 ``` 
 blockframes/libs/organization/campaign/src/lib/form/fundings/fundings.component.html

**Review by fritzschoff — CHANGES_REQUESTED** [review]

File: apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts [hunk 2]
```diff
@@ -118,6 +128,7 @@ export class MarketplaceMovieViewComponent implements OnInit {
       for (const user of users) {
         const data = {
           ...emailData,
+          currency: getCurrencySymbol(this.currency, 'wide'),
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts:131]: whydo you need that?

File: apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts [hunk 3]
```diff
@@ -91,6 +94,14 @@ export class MarketplaceMovieViewComponent implements OnInit {
     this.campaign$ = this.movieQuery.selectActiveId().pipe(
       switchMap(id => this.campaignService.valueChanges(id))
     );
+
+    this.sub = this.campaign$.subscribe(data => {
+      this.currency = data.currency;
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts:99]: do you need that?

**Review by fritzschoff — CHANGES_REQUESTED** [review]

File: apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts (hunk 2 above)
Comment (Mussuranae) [reply apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts:131]: To change the template of the email, the currency should be updated depending on the film/campaign. It's a rough data in the template for now but it should be updated after the next hotfix if we implement this commit. And we don't want a `EUR 123` but `€123` in the email.

File: apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts (hunk 3 above)
Comment (Mussuranae) [reply apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts:99]: It's related to the comment above

File: apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts [hunk 4]
```diff
@@ -91,6 +94,14 @@ export class MarketplaceMovieViewComponent implements OnInit {
     this.campaign$ = this.movieQuery.selectActiveId().pipe(
       switchMap(id => this.campaignService.valueChanges(id))
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts:95]: add a tap here so you don't need to subscribe to the campaign$ two lines below.
``` 
   this.campaign$ = this.movieQuery.selectActiveId().pipe(
      switchMap(id => this.campaignService.valueChanges(id)),
      tap(campaign => this.currency= campaing.currency)
```

**Review by fritzschoff — APPROVED** [review]

**Pull request merged**